pkg/proxy/proxy/transparent.go

```diff
@@ -16,18 +16,18 @@ import (
 	"github.com/elazarl/goproxy"
 	"github.com/google/oss-rebuild/internal/proxy/handshake"
 	"github.com/google/oss-rebuild/pkg/proxy/cert"
-	"github.com/google/oss-rebuild/pkg/proxy/logger"
+	"github.com/google/oss-rebuild/pkg/proxy/netlog"
 )
 
 // TLS port to which proxied TLS traffic should be redirected.
 // Used to enable customization during testing.
 var tlsPort = "443"
 
-// TransparentProxyServer transparently proxies HTTP and HTTPS traffic.
-type TransparentProxyServer struct {
+// TransparentProxyService transparently proxies HTTP and HTTPS traffic.
+type TransparentProxyService struct {
```

Comment:
nit: want to move this down below NewTransparentProxyServer so it's near its methods?